SECURITY.md

```
@@ -2,15 +2,14 @@
 
 | Version | Supported          |
 | ------- | ------------------ |
-| 5.2.4   | :white_check_mark: |
+| 6.0.2   | :white_check_mark: |
 
 ## Reporting a Vulnerability
 
-If you have found any security vulnerability you can contact us via 
-kim.kulling@googlemail.com
+If you have found any security vulnerability you can contact us via
+kim.kulling@assimp.org
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)